python/lsst/metaserv/schemaToMeta.py

```diff
@@ -260,21 +260,21 @@ def _retrDefaultValue(self, str):
     # "    UNIQUE UQ_AmpMap_ampName(ampName)"
     # "    UNIQUE UQ_x(xx DESC, yy),"
 
-    def _retrColumns(self, str):
-        xx = re.search(r'[\s\w_]+\(([\w ,]+)\)', str.rstrip())
-        xx = xx.group(1).split() # skip " ASC", " DESC" etc
+    def _retrColumns(self, theString):
+        values = re.search(r'[\s\w_]+\(([\w ,]+)\)', theString.rstrip())
```

Comment:
The initial part of this (`[\s\w_]+`) does very little, especially since you already know this is an index definition.  What you really want is just the text within the parentheses.
You're not properly handling strings like `ASC,`
You could rewrite this more correctly as:
```python
    columnExprs = re.search(r'\((.+?)\)', theString).group(1).split(',')
    columns = [" ".join([word for word in expr.split() if word not in ('ASC', 'DESC')])
            for expr in columnExprs]
    return ", ".join(columns)
```